### PR TITLE
Add standalone cases.

### PR DESCRIPTION
**Pull request by yzha107**

### CODE REVIEW

File: tests/standalone/standalone.sh [hunk 1]
```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```
Comment (TianlongLiang) [inline tests/standalone/standalone.sh:1]: Maybe should add a license header here?

File: tests/standalone/test-pthread/create_join.c [hunk 2]
```diff
@@ -0,0 +1,69 @@
+/*
```
Comment (xujuntwt95329) [inline tests/standalone/test-pthread/create_join.c:1]: This file is copied from https://web.dev/articles/wasm-threads, had better declare the original link

File: tests/standalone/standalone.sh (hunk 1 above)
Comment (yzha107) [reply tests/standalone/standalone.sh:1]: Yes, forgot this file. Just added.

File: tests/standalone/test-pthread/create_join.c (hunk 2 above)
Comment (yzha107) [reply tests/standalone/test-pthread/create_join.c:1]: OK. Modified the header.

**Comment by yzha107** [comment]
@xujuntwt95329, @TianlongLiang , @wenyongh , are there any other issue of this PR?

**Comment by TianlongLiang** [comment]
LGTM

**Review by wenyongh — APPROVED** [review]
LGTM

**Pull request merged**